src/routes/CategoryPage/Bestsellers.tsx

```diff
@@ -8,41 +8,44 @@ const Bestsellers = () => {
   const { category } = useParams();
   const linkArr = category ? categoryLinks[category] : [];
 
-  const highRatedProducts = getDataRatingAndDiscount(data, "discount");
+  const highRatedProducts = getDataRatingAndDiscount(
+    data,
+    "discount",
+  ) as Product[];
 
-  if (highRatedProducts)
+  if (!highRatedProducts) return null;
+  else
```

Comment:
jak masz return w if to nie potrzebujesz else. To jest właśnie cała idea szybkiego returnowania - uciekanie z zagnieżdżeń w kodzie